aetcd/client.py

```diff
@@ -533,8 +615,6 @@ async def delete_range(
             delete_response.prev_kvs,
         )
 
-    @_handle_errors
```

Comment:
@jpark-prestolabs is this intentional? The docorators are removed only for some operations, not all.

Reply:
Could you please clarify?

Reply:
I mean, `connect` is a `noop` if it already connected and waits if connecting. I'll put the docarators back. Please let me know if it breaks something.

Reply:
Since it just calls `self.transaction()` which has `_handle_errors` and `_ensure_connected` decorators.
I'd like to avoid duplication.

Reply:
Yeah, you're right, I see. But you've added `handle_errror` to `connect', it also duplicates everything.

Reply:
The only place I see `connect` undecorated is a client context manager.

Reply:
Maybe it is better to just wrap `connect` in `__aenter__` with `_handle_error`.

Reply:
`connect` should be rarely called, so I leave it as is. Thanks!